onchain/rollups/test/foundry/util/LibServerManager.sol

```diff
@@ -33,7 +33,7 @@ library LibServerManager {
 
     struct RawProof {
         bytes32 context;
-        string inputIndex;
+        string inputIndexWithinEpoch;
```

Comment:
If you take a peek in `finish_epoch_response.json`, you'll see this is indeed `inputIndex`.

Reply:
Then we should modify the `proves` function in the `LibServerManager.sol`. Instead of  checking `Proof.inputIndex`, we will be checking `Proof.validity.inputIndexWithinEpoch`.

Reply:
Yes, this makes sense.